src/content/plugins/split-chunks-plugin.md

```diff
@@ -169,6 +170,8 @@ The name of the split chunk. Providing `true` will automatically generate a name
 
 Providing a string or a function allows you to use a custom name. Specifying either a string or a function that always returns the same string will merge all common modules and vendors into a single chunk. This might lead to bigger initial downloads and slow down page loads.
 
+If you choose to specify a function, you may find the `chunk.name` and `chunk.hash` properties (where `chunk` is an element of the `chunks` array) particularly useful in choosing a name for your chunk.
```

Comment:
Could you please add an example?

Reply:
I am not sure about this, i had a case when `name` was undefined

Reply:
Maybe say that it is a NormalModule (possible not only NormalModule), and link to info about NormalModule or to NormalModule source code under webpack/webpack repo

Reply:
scratch that, it was `module` who didnt have name